### PR TITLE
Added Intagram BOT #1296

### LINKED ISSUE #1296: Instagram BOT Script
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### Describe the script

# This Instagram Script will Include the following functions:
* Download Profile Image from Username
* Download All Posts from Username
* Download Images from Hashtags

### Language

Python

### Is the feature request related to a problem? Please Specify.

_No response_

### Describe the solution you'd like.

_No response_

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by harshitbansall**
# Description

### This Python Script has the following functions:
* Download Profile Image from Username
* Download All Posts from Username
* Download Images from Hashtags

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Instagram_BOT/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# 🍕 Instagram BOT - [Harshit Bansal](https://github.com/harshitbansall)
```
Comment (HarshCasper) [inline Python/Instagram_BOT/README.md:1]: Removed the username from the title

File: Python/Instagram_BOT/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Prerequisites
```
Comment (HarshCasper) [inline Python/Instagram_BOT/README.md:2]: Give a line break

File: Python/Instagram_BOT/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```
+* Activate the Environment
+```
+env\Scripts\activate
+```
+* Install all the dependencies
+```
+pip install -r requirements.txt
+```
+## Usage
```
Comment (HarshCasper) [inline Python/Instagram_BOT/README.md:15]: Give a line break

File: Python/Instagram_BOT/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+</div>
+<br>
+
+* <code>Example 3</code> - Download Hash Tag Images of <code>dogs</code>
+
+<div align = "center">
+<img src = "https://user-images.githubusercontent.com/24206423/156178673-6dc13f7b-49d6-4284-8cbd-9310356ad300.gif">
+</div>
+
+### If you liked this script and and used it, Please follow [Harshit Bansal](https://github.com/harshitbansall).
```
Comment (HarshCasper) [inline Python/Instagram_BOT/README.md:45]: Just add an author section with a link to your profile. If someone is looking to follow, they would do it without being prompted.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Looks pretty good to me.

Change the name of Python script to `instagram_bot.py`.

**Comment by harshitbansall** [comment]
@HarshCasper I've applied the changes you requested.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**